script/install-openldap

```diff
@@ -48,20 +48,20 @@ chown -R openldap.openldap /var/lib/ldap
 rm -rf $TMPDIR
 
 # SSL
+export CA_CERT="/etc/ssl/certs/cacert.pem"
+export CA_KEY="/etc/ssl/private/cakey.pem"
+export CA_INFO="/etc/ssl/ca.info"
 
-sh -c "certtool --generate-privkey > /etc/ssl/private/cakey.pem"
+# If you ever need to regenerate these...
+# certtool --generate-privkey > /path/to/cakey.pem
+# certtool --generate-self-signed \
+#  --load-privkey /path/to/cakey.pem
+#  --template /path/to/ca.info
+#  --outfile /path/to/cacert.pem
```

Comment:
Yes, there was a previous fixture CA, removed in this PR in caf191102f5d04cb1e4222b6b75c15a44470134e. Oddly, it was just the CA cert, no private key. Travis ignored it completely and generated its own self-signed cert/key pair on every run. Because the Vagrant tests execute on a different host, it's easier to use a fixture keypair burned into the repo. 

I left the commented-out lines in there in case the fixture CA needs to be regenerated, or in case future custom TLS tests require specific cert values. We could move the comment out into `script/generate-fixture-ca` or something if you like.